appveyor.yml

```diff
@@ -8,6 +8,7 @@ platform:
 install:
   - ps: Install-Product node $env:nodejs_version
   - npm install --production
+  - npm install sinon@7.1.1 npm-which@3.0.1 object-to-spawn-args@1.1.1 jest@21.2.1 conventional-changelog-cli@1.3.2 
```

Comment:
Why do we have to do this? If you need dev dependencies, just change the line above and remove `--production`